core/src/main/java/org/geysermc/geyser/translator/inventory/item/ItemTranslator.java

```diff
@@ -520,6 +526,19 @@ public static CompoundTag translateDisplayProperties(GeyserSession session, Comp
         return tag;
     }
 
+    /**
+     * Translates the custom model data of an item
+     */
+    public static ItemData.Builder translateCustomModelData(CompoundTag nbt, ItemData.Builder builder, ItemMapping mapping) {
+        if (nbt != null && nbt.get("CustomModelData") != null) {
```

Comment:
No need to call `get` twice here. Just set it as a variable and add another if statement to check if it's an instance of IntTag.